frontend/public/components/RBAC/bindings.jsx

```diff
@@ -131,30 +162,35 @@ export const RoleLink = ({binding}) => {
   return <ResourceLink kind={kind} name={binding.roleRef.name} namespace={ns} />;
 };
 
-const Row = ({obj: binding}) => <ResourceRow obj={binding}>
-  <div className="col-md-3 col-sm-4 col-xs-6">
-    <ResourceLink kind={bindingKind(binding)} name={binding.metadata.name} namespace={binding.metadata.namespace} className="co-resource-item__resource-name" />
-  </div>
-  <div className="col-md-3 col-sm-4 hidden-xs co-break-word">
-    <RoleLink binding={binding} />
-  </div>
-  <div className="col-md-2 hidden-sm hidden-xs co-break-word">
-    {binding.subject.kind}
-  </div>
-  <div className="col-md-2 hidden-sm hidden-xs co-break-word">
-    {binding.subject.name}
-  </div>
-  <div className="col-md-2 col-sm-4 col-xs-6 co-break-word">
-    {binding.metadata.namespace ? <ResourceLink kind="Namespace" name={binding.metadata.namespace} /> : 'all'}
-  </div>
-  <div className="dropdown-kebab-pf">
-    <BindingKebab binding={binding} />
-  </div>
-</ResourceRow>;
+const RoleBindingsTableRow = ({obj: binding, index, key, style}) => {
+  return (
+    <Vr id={binding.metadata.uid} index={index} trKey={key} style={style}>
+      <Vd className={tableColumnClasses[0]}>
+        <ResourceLink kind={bindingKind(binding)} name={binding.metadata.name} namespace={binding.metadata.namespace} className="co-resource-item__resource-name" />
+      </Vd>
+      <Vd className={classNames(tableColumnClasses[1], 'co-break-word')}>
+        <RoleLink binding={binding} />
+      </Vd>
+      <Vd className={classNames(tableColumnClasses[2], 'co-break-word')}>
+        {binding.subject.kind}
+      </Vd>
+      <Vd className={classNames(tableColumnClasses[3], 'co-break-word')}>
+        {binding.subject.name}
+      </Vd>
+      <Vd className={classNames(tableColumnClasses[4], 'co-break-word')}>
+        {binding.metadata.namespace ? <ResourceLink kind="Namespace" name={binding.metadata.namespace} /> : 'all'}
+      </Vd>
+      <Vd className={tableColumnClasses[5]}>
+        <BindingKebab binding={binding} />
+      </Vd>
+    </Vr>
+  );
+};
+RoleBindingsTableRow.displayName = 'RoleBindingsTableRow';
 
 const EmptyMsg = () => <MsgBox title="No Role Bindings Found" detail="Roles grant access to types of objects in the cluster. Roles are applied to a group or user via a Role Binding." />;
 
-export const BindingsList = props => <List {...props} EmptyMsg={EmptyMsg} Header={Header} Row={Row} />;
+export const BindingsList = props => <Table {...props} aria-label="Role Bindings" EmptyMsg={EmptyMsg} Header={RoleBindingsTableHeader} Row={RoleBindingsTableRow} virtualize />;
```

Comment:
Doesn't this `aria-label` repeat what's in the header? Won't screen readers read the same heading twice?

Reply:
this is the attribute passed to the `role=grid` element, and that is where `aria-label` should be added for screen readers to correctly announce them.